spec/factories/comments.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :comment do
+    body Faker::Lorem.paragraph
+    user
```

Comment:
what is this? are you creating an user or not?
Put all values inside {}, like body { Faker::Lorem.paragraph }